godtools/App/Share/Domain/UseCases/GetMobileContentAuthTokenUseCase/GetMobileContentAuthTokenUseCase.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  GetMobileContentAuthTokenUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 11/7/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import OktaAuthentication
+import Combine
+
+class GetMobileContentAuthTokenUseCase {
```

Comment:
Here we don't need this UseCase since there isn't any business rules being applied.  The ```AuthenticateUserUseCase``` should communicate directly with the ```AuthTokenRepository``` and there should always be a new auth token fetched when a user authenticates.